package.json

```diff
@@ -47,13 +45,8 @@
     "ts-node": "^10.9.1",
     "typescript": "^4.1.3"
   },
-  "config": {
-    "commitizen": {
-      "path": "cz-conventional-changelog"
-    }
-  },
```

Comment:
we did this in dev-scripts: https://github.com/forcedotcom/dev-scripts/pull/92
I manually removed these, didn't want to bump it to v3 now because of the linter changes required.